vignettes/multiple-factors.Rmd

```diff
@@ -0,0 +1,196 @@
+---
+title: "Issue of Multi-Factor Measurement Model"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{multiple-factors}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r, message = FALSE}
+library(lavaan)
+library(R2spa)
+```
+
+The example is from https://lavaan.ugent.be/tutorial/sem.html. 
+
+# Factor score
```

Comment:
should we use ## so that the header font size can be distinguished from the title size?

Reply:
Sure